internal/store/postgres/record_test.go

```diff
@@ -0,0 +1,47 @@
+package postgres_test
+
+import (
+	"fmt"
+	"strings"
+
+	"gorm.io/gorm"
+)
+
+func readStoredRecords[R any](db *gorm.DB) []R {
```

Comment:
yes, it is indeed a utility function used for unit testing only. at first, I wanted to put it under `postgres_test.go`, since we don't have any unit test specific there, and only some utilities. I didn't put it there because I thought `postgres_test.go` is dedicated to setting up a fresh database. and since I want to avoid utils, as much as possible, I ended up using `record_test.go`.

if we don't want to use the current one, then, what do you think we rename the file name to `utils_record_test.go`?